posthog-core/src/gzip.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Older browsers and some runtimes don't support this yet
+ */
+export function isGzipSupported(): boolean {
+  return 'CompressionStream' in globalThis
+}
+
+/**
+ * Gzip a string using Compression Streams API if it's available
+ */
+export async function gzipCompress(input: string, isDebug = true): Promise<Blob | null> {
+  if (!isGzipSupported()) {
+    return null
+  }
```

Comment:
not needed since we already do `this.disableCompression = !isGzipSupported() || (options?.disableCompression ?? false)`, and we only call `gzipCompress` if `disableCompression` is false

Reply:
Ah, this is from the pre try-catch. I'll remove it